content/guides/end-to-end-testing/amazon-cognito-authentication.md

```diff
@@ -157,6 +158,169 @@ const awsConfig = require('./aws-exports-es5.js')
 
 ## Custom Command for Amazon Cognito Authentication
 
+There are two ways you can authenticate to AWS Cognito
+
+- [Login with cy.origin()](/guides/end-to-end-testing/amazon-cognito-authentication#Login-with-cy-origin)
+- [Legacy Programmatic Access](/guides/end-to-end-testing/amazon-cognito-authentication#Programmatic-Login-Legacy)
+
+### Login with [`cy.origin()`](/api/commands/origin)
+
+Next, we'll write a command to perform a login to
+[Amazon Cognito](https://aws.amazon.com/cognito) using
+[`cy.origin()`](/api/commands/origin) to navigate to the cognito origin,
+inputting a user credentials, and signing in via OAuth to redirect back to our
```

Comment:
```suggestion
[`cy.origin()`](/api/commands/origin) to:
1.  navigate to the cognito origin
2. inputting a user credentials
3. signing in via OAuth which will redirect back to our application
4. cache the results with
[`cy.session()`](/api/commands/session).
```
just an idea. I know when I'm learning, I only skim content instead of read in depth

Reply:
addressed in e14bf7170